src/components/Section/Relationships/RelationshipStatus/CivilUnion.test.jsx

```diff
@@ -4,8 +4,9 @@ import configureMockStore from 'redux-mock-store'
 import { Provider } from 'react-redux'
 import CivilUnion from './CivilUnion'
 
-const mountComponent = (mockStore, Component, props) => {
-  const store = mockStore({ application: { AddressBooks: {} }})
+const mountComponent = (Component, props = {}) => {
```

Comment:
Why is this called mountComponent and the others are call createComponent? I'd recommend a consistent name if possible for this function in all the tests.

Reply:
Well, while they perform similar functions, these 2 different functions have different function signatures – `mountComponent` takes the component as an argument, `createComponent` does not.

While the 3 components that use `mountComponent` could be refactored further, the current PR seems to make a good tradeoff between changing the bare minimum required to the existing tests to make the `DateControl` change pass and more sweeping changes across the entire front-end test suite.